openstack/glance/templates/etc/_glance-policy.json.tpl

```diff
@@ -1,53 +1,63 @@
 {
-    "context_is_cloud_admin":  "role:cloud_image_admin",
-    "context_is_admin":  "rule:context_is_cloud_admin",
-    "default": "rule:context_is_admin",
```

Comment:
Please don't remove the default rule. Everything that is not explicitly stated falls back to default and should only be allowed for `context_is_admin`
I.e. put it back in like it was before, please